cmake/GoogleCloudCppLibrary.cmake

```diff
@@ -201,7 +208,16 @@ function (google_cloud_cpp_add_ga_grpc_library library display_name)
 
     # Create and install the CMake configuration files.
     include(CMakePackageConfigHelpers)
-    configure_file("config.cmake.in" "${library_target}-config.cmake" @ONLY)
+    set(GOOGLE_CLOUD_CPP_CONFIG_LIBRARY "${library_target}")
+    unset(find_dependencies)
```

Comment:
as above.

Reply:
Done, but I am skeptical as to whether this one was an improvement. I briefly explored `list(TRANSFORM ... REPLACE ...)`, but gave up.